provision/vvv-init.sh

```diff
@@ -209,10 +209,10 @@ setup_cli() {
   echo "# auto-generated file" > "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "path: \"${PUBLIC_DIR}\"" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "@vvv:" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
-  echo "  ssh: vagrant@${DOMAIN}" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
+  echo "  ssh: vagrant" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "  path: ${PUBLIC_DIR_PATH}" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
   echo "@${VVV_SITE_NAME}:" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
-  echo "  ssh: vagrant@${DOMAIN}" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
+  echo "  ssh: vagrant" >> "${VVV_PATH_TO_SITE}/wp-cli.yml"
```

Comment:
These changes cripple WP CLI functionality on all environments. It also does not make sense as a change, we don't add a `vagrant` domain  to the hosts  file, so how would it know where the VM is?